ergo/platforms/almanis.py

```diff
@@ -0,0 +1,226 @@
+from abc import abstractmethod
+import json
+from typing import Dict, List, Sequence
+
+import requests
+
+from ergo.questions import BinaryQuestion, Question
+
+
+class Almanis:
+    """
+    The main class for interacting with Almanis
+    """
+
+    question_id_request_json = json.dumps(
+        {
+            "customerID": "44d7f0b3-7e6f-4867-b00a-6816492bb510",
+            "statuses": [{"Trading": {}}, {"ClosedForTrading": {}}],
+        }
+    )
+
+    BINARY_QUESTION_KEY = "BooleanQuestionOD"
+    CONTINUOUS_QUESTION_KEY = "MultipleChoiceQuestionOD"
+
+    def __init__(self):
+        self.api_url = "https://proddysruptlabsr53tobackendapi.dysruptlabs.com/api/sharedCode/api/IntegrationLayerApi"
+        self.s = requests.Session()
+        self.question_ids = []
+
+    def refresh_question_ids(self):
+        response = self.s.post(
+            f"{self.api_url}/getQuestionIDs", data=self.question_id_request_json
+        )
+        response_json = response.json()
+        self.question_ids = [x["id"] for x in response_json]
+
+    def get_questions(self, ids: List[str]) -> Sequence["AlmanisQuestion"]:
```

Comment:
Does including `"`s in the type hints work? I would think it should just be e.g. `Sequence[AlmanisQuestion]` but maybe both work

Reply:
Yeah, I tried it without `"`s at first, but apparently Python type hinting doesn't work for a class you're declaring in the same file. You can get around this by using [postponed evaluation](https://www.python.org/dev/peps/pep-0563/#enabling-the-future-behavior-in-python-3-7), but that only works in Python 3.7 which it looks like Ergo doesn't support yet (https://github.com/oughtinc/ergo/issues/412)